src/indexApi.ts

```diff
@@ -1,13 +1,14 @@
 // BASE SETUP
 // =============================================================================
 
-import { asObject, asString } from 'cleaners'
+import { asObject, asOptional, asString } from 'cleaners'
 import cluster from 'cluster'
 import { formatISO } from 'date-fns'
 import { forkChildren, rebuildCouch } from 'edge-server-tools'
 
 import { config } from './utils/config'
 import { couchSchema, minAmtDbPrefix } from './utils/couchSchema'
+import { ErrorResponse, makeErrorResponse } from './utils/errorResponse'
```

Comment:
You should reorganize your commits so that the errorResponse file is created before the minimum amount Endpoint is created.  Right now if you load commit by commit, the endpoint commit try to import something that doesn't exist.

Reply:
Good catch. I have changed the order per your comment.